### PR TITLE
Store the risk outputs in the datastore

### PR DESCRIPTION
**Pull request by micheles**
For the moment I am fixing the event based calculator and the oq-lite command pick_rlzs, plot and show.
Other pull requests will follow to fix the other calculators.
The tests are running here: https://ci.openquake.org/job/zdevel_oq-risklib/524

### CODE REVIEW

File: openquake/commonlib/commands/plot.py [hunk 1]
```diff
@@ -52,57 +51,51 @@ def make_figure(indices, imtls, spec_curves, curves=(), label=''):
...
         [curves_by_rlz[rlz] for rlz in rlzs], weights)
 
 
-def plot(hazard_pik, hazard_pik2=None, sites='0'):
+def plot(calc_id, other_id=None, sites='0'):
     """
     Hazard curves plotter.
-
-    :param hazard_pik: the pathname to a pickled file
-    :param hazard_pik2: None or the pathname to another pickled file
```
Comment (ptormene) [inline openquake/commonlib/commands/plot.py]: Please add the new params

File: openquake/commonlib/commands/show.py [hunk 2]
```diff
@@ -16,19 +16,52 @@
...
+        nbytes /= 1024.
+        i += 1
+    f = ('%.2f' % nbytes).rstrip('0').rstrip('.')
+    return '%s %s' % (f, suffixes[i])
+
+
+def show(calc_id, key=None, rlzs=None):
+    """
+    Show the content of a datastore
+    """
```
Comment (ptormene) [inline openquake/commonlib/commands/show.py]: Please describe the params in the docstring

**Comment by ptormene** [comment]
Apart from the minor comments, LGTM

File: openquake/commonlib/commands/plot.py (hunk 1 above)
Comment (micheles) [inline openquake/commonlib/commands/plot.py]: Added

File: openquake/commonlib/commands/show.py (hunk 2 above)
Comment (micheles) [inline openquake/commonlib/commands/show.py]: Done

**Pull request merged**